### ISSUE TITLE
Use of virtualenv (for Py3) leads to venv creating broken venvs

### ISSUE THREAD

**Issue by ulope** [mitsuhiko/pipsi#160]
Pipsi always uses `virtualenv` (even on Python 3) to create the virtualenvs packages are installed into.

Normally this isn't a problem, however there seems to an incompatibility with `venv` when running inside a `virtualenv` virtualenv. 

Related issues: 
- https://github.com/pypa/virtualenv/issues/1095
- https://bugs.python.org/issue30811

This is unfortunate since it causes tools installed with pipsi being unable to create other virtualenvs using the builtin `venv` module.

A solution would be to use `venv` instead of `virtualenv` on Python3.

**Comment by ramiro** [comment]
Seems to be duplicate of #123 and #142 and already fixed in master.

**Comment by anowlcalledjosh** [comment]
Indeed, appears to be fixed (using b43229d8a to test): after reinstalling Poetry via pipsi, `poetry install` now creates a working virtualenv.

**Comment by GreenGremlin** [comment]
Any word on when the fix might be released?

**Comment by cs01** [comment]
pipsi is no longer maintained. You may have better luck with [pipx](https://github.com/pipxproject/pipx).
